main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/urfave/cli"
+)
+
+var VERSION = "v0.0.0-dev"
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Rancher 1.6 to Rancher 2.0 migration-helper"
+	app.Version = VERSION
+	app.Usage = "Please check the options using --help flag"
+	app.Action = func(c *cli.Context) error {
+		StartTool(c)
+		return nil
+	}
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "docker-file",
+			Usage: "Specify absolute path to an alternate Docker compose file (default: docker-compose.yml)",
```

Comment:
Nit: usage should just describe the purpose of the parameter, not the action. Given that we are going to provide the default value, it can be rephrased to something like `An absolute path to a Docker compose file`